operator/channels/packages/configconnector/1.113.0/namespaced/per-namespace-components.yaml

```diff
@@ -168,24 +164,6 @@ spec:
           privileged: false
           runAsNonRoot: true
           runAsUser: 1000
-      - command:
-        - /monitor
-        - --source=configconnector:http://localhost:8888?whitelisted=reconcile_requests_total,reconcile_request_duration_seconds,reconcile_workers_total,reconcile_occupied_workers_total,internal_errors_total&customResourceType=k8s_container&customLabels[container_name]&customLabels[project_id]&customLabels[location]&customLabels[cluster_name]&customLabels[namespace_name]=$(POD_NAMESPACE)&customLabels[pod_name]=$(POD_NAME)
-        - --stackdriver-prefix=kubernetes.io/internal/addons
-        env:
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        image: gke.gcr.io/prometheus-to-sd:v0.11.12-gke.0
-        name: prom-to-sd
```

Comment:
Is removing "prom-to-sd" intentional?

Reply:
Good catch - and @gemmahou caught this also!  We think it was caused by a newer version of kustomize; we should make sure that `make upgrade-kcc` (and everything) runs with a known version of kustomize

Reply:
( automated PR also staging removal: [ref](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1142/files#diff-893e90058739be2dda3bb215a7605399edada7ef4a914c5d2013ad0335ee5d3fL185-L188)